vendor.conf

```diff
@@ -27,7 +27,7 @@ golang.org/x/net 7dcfb8076726a3fdd9353b6b8a1f1b6be6811bd6
 google.golang.org/grpc v1.3.0
 github.com/pkg/errors v0.8.0
 github.com/opencontainers/go-digest 21dfd564fd89c944783d00d069f33e3e7123c448
-golang.org/x/sys f3918c30c5c2cb527c0b071a27c35120a6c0719a
+golang.org/x/sys fb4cac33e3196ff7f507ab9b2d2a44b0142f5b5a https://github.com/golang/sys
```

Comment:
Possibly a dumb question, but why is the URL needed here? Removing it a rerunning `vndr` doesn't produce any changes.